src/panels/config/integrations/integration-panels/zha/zha-config-dashboard.ts

```diff
@@ -32,6 +37,11 @@ export const zhaTabs: PageNavigation[] = [
     path: `/config/zha/groups`,
     iconPath: mdiFolderMultipleOutline,
   },
+  {
+    translationKey: "ui.panel.config.zha.visualization.caption",
+    path: `/config/zha/visualization`,
+    iconPath: mdiApacheKafka,
```

Comment:
We should not use a brand logo for this, I guess the lan might be a little bit suited? I can't find a good mesh network icon...
```suggestion
    iconPath: mdiLan,
```

Reply:
I had suggested Kafka 😬